astropy/io/fits/convenience.py

```diff
@@ -383,7 +383,7 @@ def delval(filename, keyword, *args, **kwargs):
 
 
 def writeto(filename, data, header=None, output_verify='exception',
-            clobber=False, checksum=False):
+            clobber=None, overwrite=None, checksum=False):
```

Comment:
Neither, since otherwise it is not possible to test for inconsistency, when somehow both clobber and overwrite are used (but with inconsistent values).